<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Japanese Verb Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="layout">
    <item row="0" column="2">
     <widget class="QLineEdit" name="searchBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="searchLabel">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListView" name="filesView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="HighlightTextEdit" name="textView">
       <property name="font">
        <font>
         <family>Yu Gothic</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenDir"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionRealtimeSearch"/>
    <addaction name="actionRegExpSearch"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenDir">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open working directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRealtimeSearch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search: Realtime</string>
   </property>
  </action>
  <action name="actionRegExpSearch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search: Use RexExp engine</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HighlightTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>jvs/highlighttextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchBox</tabstop>
  <tabstop>filesView</tabstop>
  <tabstop>textView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
